docs/workflow/blocks/mapping.rst

```diff
@@ -72,19 +76,19 @@ Output will be:
   surname: "Doe"
   iAmNull: null
 
-We've seen that property values of object keys can be set using backticks, and it is also possible to wrap a whole JSON object in backticks. 
-The output will be the same as this:
+Is also possible to wrap a whole JSON object in backticks:
```

Comment:
Minor syntax change -

"It is also possible to wrap a whole JSON object in backticks:"